docs/src/tutorials/huggingface.rst

```diff
@@ -0,0 +1,208 @@
+*****************************************************************************************
+Hyperparameters optimisation using a HuggingFace Model and the Hydra-Orion-Sweeper plugin
+*****************************************************************************************
+
+In this tutorial, we will show an easy Orion integration of a HuggingFace translation model using
+Hydra, with the `Hydra_Orion_Sweeper <https://github.com/Epistimio/hydra_orion_sweeper>`_ plugin.
+Hydra is essentially a framework for configuring applications. We will use it to define our
+Hyperparameters and some Orion configuration. We will also be using
+`Comet <https://www.comet.com/>`_ for monitoring our experiments.
+Installation
+^^^^^^^^^^^^
+For this tutorial everything that we need to download can be can be found in the ``requirements.txt``
+file located in the ``examples/huggingface`` repository.
+
+.. include:: /../../examples/huggingface/requirements.txt
```

Comment:
Instead of listing the file's content, I think it would be more useful to say how to install with `pip`: 
```bash
$ pip install -r examples/huggingface/requirements.txt
```

Reply:
Fixed